tests/data/rxnorm/rxnorm_2022-12-05.RRF

```diff
@@ -0,0 +1,663 @@
+44|ENG||||||10283027||||USP|IN|m49500|Mesna||N||

```

Comment:
Does the license allow us to share RxNorm data publicly?

Reply:
This is a good question and I'll probably have to double check w/ alex

Reply:
I went ahead and pared down the file to just the rows in the permitted sources constant. I think that's a good enough middle ground.